chain/network/src/routing/routing_table_actor.rs

```diff
@@ -326,8 +329,10 @@ impl RoutingTableActor {
         let edges_to_remove = self
             .edges_info
             .iter()
-            .filter_map(|(key, edge)| {
-                if peers_to_remove.contains(&key.0) || peers_to_remove.contains(&key.1) {
+            .filter_map(|edge| {
```

Comment:
unrelated tho the PR, but I'd write this as `.filter(||).cloned()` rather than as a `.filter_map`

Reply:
@matklad This whole function should be rewritten. However, I'm waiting with refactor after this PR gets shipped.